simbad/command_line/simbad_database.py

```diff
@@ -143,7 +145,15 @@ def create_lattice_db(database):
             'unitCellAngleGamma,spaceGroup,experimentalTechnique&service=wsfile&format=csv'
 
     crystal_data, error_count = [], 0
-    for line in urllib2.urlopen(url):
+
+    rcsbReportFile=os.path.join(os.environ["CCP4_SCR"], "rcsb" + "_%s.csv" % str(datetime.datetime.now()).replace(" ","_").replace("-","_").replace(":","_").replace(".","_")) 
+    urllib.urlretrieve(url, rcsbReportFile)
```

Comment:
I'd suggest/recommend the use of [`requests`](https://2.python-requests.org/en/master/) instead of `urllib`